### ISSUE TITLE
OpenAPI spec is missing properties for results array for /task endpoint

### ISSUE THREAD

**Issue by mstokes-f5** [F5Networks/f5-appsvcs-extension#320]
### Environment
 * Application Services Version: 3.21.0-4
 * BIG-IP Version: N/A

### Summary
The [OpenAPI docs for the /task endpoint on AS3](https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/refguide/apidocs.html#tag/Task) is missing the following fields for the `results` object:
* response
* errors
* declarationFullId

Also, it has a `messagee` that I have never seen, and I suspect is a typo for `message`.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
  "class": "ADC",
  "schemaVersion": "3.0.0",
  "foo": {
    "class": "Tenant",
    "bar": {
      "class": "Application",
      "template": "udp",
      "serviceMain": {
        "class": "Service_UDP",
        "virtualAddresses": [
          "192.0.2.1"
        ],
        "virtualPort": 5555,
        "pool": "bar_Pool1"
      },
      "bar_Pool1": {
        "class": "Pool",
        "monitors": [
          "icmp"
        ],
        "members": [
          {
            "serverAddresses": null,
            "servicePort": 5555
          }
        ]
      }
    }
  }
}
```

2. Observe the following task (gathered via a GET to `/task`):
```json
    {
      "id": "e538ed77-f9d1-42e0-8f6b-ae085a31eb47",
      "results": [
        {
          "code": 422,
          "errors": [
            "/foo/bar/bar_Pool1/members/0: should have required property 'bigip'"
          ],
          "declarationFullId": "",
          "message": "declaration is invalid"
        }
      ],
      "declaration": {}
    }
```

**Comment by dstokesf5** [comment]
Added to our internal product backlog as AUTOTOOL-1793.

**Comment by dstokesf5** [comment]
This was resolved with the 3.31.0 release of AS3.
